mmv1/compiler.rb

```diff
@@ -109,6 +109,7 @@
   # This writes to a fake demo product currently. In the future this should
   # produce the entire product folder including product.yaml for a single OpenAPI spec
   OpenAPIGenerate::Parser.new('openapi_generate/openapi/*', 'products').run
+  return
```

Comment:
Oh weird, was this not implicit?

Reply:
Oh Ruby- the implicit behaviour was returning the results of `.run` above

Reply:
It's just to short-circuit when running the generator. No need to wait for the output of mmv1 run if we need to add examples by hand anyways